onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -494,6 +496,12 @@ void ResizeBiCubic(
     }
   }
 
+  // setup up temp arrays to hold coefficients when exclude_outside is set to true
+  std::array<float, CubicModeGridLength> y_coeff_holder;
+  std::array<float, CubicModeGridLength> x_coeff_holder;
+  float y_coeff_sum = 1;
+  float x_coeff_sum = 1;
```

Comment:
Do we want to declara these two inside the loop so the initialization is meaningful when they are first used?

Reply:
When exclude_outside is false this never ends up being changed

Reply:
I don't see much difference in either of the 2 ways...